### DISCUSSION TITLE
.todotxt file is not synced at all

### DISCUSSION THREAD

**Discussion by gerroon** [vrtmrz/obsidian-livesync#362] [Q&A]
Hi

I am using an Obsidian plugin for todotxt, it uses .todotxt extention for its own format, just  abasic txt file . Livesync does not want to sync the changes :( It syncs the initial file but no changes are updated. Is there a limitation with file extensions?

**Comment by dlphnkck** [comment]
Settings for Self-hosted LiveSync -> 🔁Sync Settings -> Synchronization target filters ->
[![Image from Gyazo](https://i.gyazo.com/33bc1875b3f06ec09a30b8e65a0f29a8.png)](https://gyazo.com/33bc1875b3f06ec09a30b8e65a0f29a8)

**Reply by gerroon** [reply]
thanks but I do not have anything set in there. It is just empty like yours. And i am not ignoring any files
